### PR TITLE
update postgres and tokio-postgres

### PR DESCRIPTION
**Pull request by AtsukiTak**
Thanks for providing this crate!

`postgres` [0.18.1](https://crates.io/crates/postgres/0.18.1) and `tokio-postgres` [0.6.0](https://crates.io/crates/tokio-postgres/0.6.0) are released recently.
It would be straightforward. Please update 😄

### CODE REVIEW

**Comment by jxs** [comment]
Hi, and thanks! Sorry I had to close and re-open again to trigger the CI pipeline which stalled. 
Hum Tests seem to be failing

**Comment by AtsukiTak** [comment]
Oh dear, that's not straightforward at all 😭 

The problem is that `tokio-postgres` depends on `tokio` v0.3 but `mysql_async` depends on v0.2.
I've updated `tokio` crate to 0.3 and imported `tokio-compat-02` crate to pass `mysq_async` tests.

But sadly, `tokio-compat-02` seems not to be compilable by [this issue](https://github.com/LucioFranco/tokio-compat-02/pull/8). For now, `tokio-compat-02` crate points GitHub directly but we need to fix it when a new version is released.

**Comment by Darksonn** [comment]
You can also add the feature yourself since features are merged across all uses of the crate.

**Comment by AtsukiTak** [comment]
@Darksonn Thanks for comment! And also thanks for releasing a new version of `tokio-compat-02` !

> You can also add the feature yourself since features are merged across all uses of the crate.

I didn't know that, thanks 😄  But now, I'm confusing since we enabled the `full` feature of `tokio` crate [in the refinery_core crate](https://github.com/rust-db/refinery/blob/9c5b4f0f5e2382f3f0631d9ff22d5bee6f3c82f7/refinery_core/Cargo.toml#L35) which is used by `refinery` crate. Maybe I missed something. Would you like to tell me the reason why we got that error?

**Comment by Darksonn** [comment]
The tokio-compat-02 crate uses the stream feature of Tokio 0.3, not of 0.2.

File: refinery_core/Cargo.toml [hunk 1]
```diff
@@ -27,12 +27,12 @@ url = "2.0"
...
 rusqlite = {version = ">= 0.23, < 0.25", optional = true}
-postgres = {version = "0.17", optional = true}
+postgres = {version = "0.18", optional = true}
 mysql = {version = "18", optional = true}
-tokio-postgres = { version = "0.5", optional = true }
+tokio-postgres = { version = "0.6", optional = true }
 mysql_async = { version = "0.25", optional = true }
 
-tokio = { version = "0.2", features = ["full"], optional = true }
+tokio = { version = "0.3", features = ["full"], optional = true }
```
[the remark this answers is not in the document]
Comment (Darksonn) [reply refinery_core/Cargo.toml:35]: I see it's marked optional? If it isn't enabled, it wouldn't have an effect.

**Comment by jxs** [comment]
LGTM thanks!

File: refinery_core/Cargo.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (AtsukiTak) [reply refinery_core/Cargo.toml:35]: It's enabled by refinery crate (for which we did test)

**Pull request merged**